### PR TITLE
bpo-35116: Adding Doc/library entries for cgi.FieldStorage max_num_fields

### PR DESCRIPTION
**Pull request by matthewbelisle-wf**
https://bugs.python.org/issue35116

### CODE REVIEW

**Comment by matthewbelisle-wf** [comment]
For your review @vstinner from your comment in https://github.com/python/cpython/pull/9969#issuecomment-434344907.

File: Doc/library/urllib.parse.rst
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: Please replace "param" with "parameter.

File: Doc/library/urllib.parse.rst
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: Ditto.

**Comment by matthewbelisle-wf** [comment]
Ditto, fixed in 1c3f2f0 @vstinner

File: Doc/library/urllib.parse.rst
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: I would just say "The optional argument *max_num_fields* is the maximum number of ..."

Note: wait until this PR is merged before updating the 3 other PRs :-)

File: Doc/library/urllib.parse.rst
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: "Default is *None*." IMHO it's redundant with the function prototype and can be removed.

File: Doc/library/urllib.parse.rst
Comment (vstinner) [inline Doc/library/urllib.parse.rst]: Ditto.

**Comment by matthewbelisle-wf** [comment]
Sure thing @vstinner fixed in 51d6fb37bf.

**Review by vstinner — APPROVED** [review]
LGTM.

**Comment by vstinner** [comment]
You can now update the 3 other PRs :-)

**Pull request merged**
